fw/makefile

```diff
@@ -6,12 +6,12 @@ ARCH = AVR8
 BOARD = BOARD_NONE
 F_CPU = 16000000
 F_USB = $(F_CPU)
-OPTIMIZATION = s
+OPTIMIZATION = 2
 TARGET = beef
 FASTLED_SRC = FastLED/src
 SRC = $(wildcard *.c) $(wildcard *.cpp) $(FASTLED_SRC)/colorutils.cpp $(FASTLED_SRC)/FastLED.cpp $(FASTLED_SRC)/lib8tion.cpp $(LUFA_SRC_USB) $(LUFA_SRC_USBCLASS)
 LUFA_PATH = LUFA
-CC_FLAGS = -DUSE_LUFA_CONFIG_HEADER -IConfig/ -include fastled_shim.h -DFASTLED_NO_PINMAP
+CC_FLAGS = -DUSE_LUFA_CONFIG_HEADER -IConfig/ -include fastled_shim.h -DFASTLED_NO_PINMAP -DBEEF_LED_REFRESH=40 -DBEEF_TT_RATIO=2
```

Comment:
`BEEF_LED_REFRESH ?= 40` would allow us to set these values from CLI without having to edit the `makefile`. Similarly for `BEEF_TT_RATIO`.

Also 50 might be a better default so that it aligns with the spin effect.